src-tauri/src/node/operation.rs

```diff
@@ -48,21 +47,13 @@ impl From<CalendarId> for Hash {
     }
 }
 
-impl From<CalendarId> for LogId {
-    fn from(id: CalendarId) -> Self {
-        LogId(id.0)
-    }
-}
-
-impl From<Hash> for LogId {
-    fn from(value: Hash) -> Self {
-        LogId(value)
-    }
-}
-
 #[derive(Clone, Debug, PartialEq, Eq, StdHash, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
-pub struct LogId(Hash);
+pub struct LogId {
+    // @TODO(adz): Currently we store everything in one log per calendar, later we want to list all
+    // possible "log types" here, for example for all events, resources, messages etc.
+    pub calendar_id: CalendarId,
+}
```

Comment:
I've refactored `LogId` slightly to make it explicit that it needs the calendar id, also it prepares the struct for the future where more fields are required (resource type and id etc.).

Reply:
It's cool to see how the log id can be "derived" from other data, so the log id itself doesn't really "exist" explicitly in the header! It only used for storage and sync